Code/Engine/Foundation/Types/Implementation/DelegateHelper_inl.h

```diff
@@ -1,25 +1,63 @@
 
-template <typename R EZ_COMMA_IF(ARG_COUNT) EZ_LIST(typename ARG, ARG_COUNT)>
```

Comment:
We could get rid of this additional inl file now and move everything into Delegate_inl.h

Reply:
Yes, I'll do that in a separate PR that will not have any functional changes.